packages/web/docs/next.config.js

```diff
@@ -240,7 +242,9 @@ export default withGuildDocs({
       permanent: true,
     },
   ],
-  swcMinify: true,
+  env: {
+    SITE_URL: 'https://the-guild.dev/graphql/hive',
```

Comment:
Won't that affect the previews?

Reply:
nope, this env is used to generate `og:url` and `<link rel="canonical"` under the hood in guild/components